### PR TITLE
Change the type of vector attributes in gen_doc.py

### PR DESCRIPTION
**Pull request by tungld**
It seems someone manually edited `op_build_table.inc` for changing vector attribute's type from `vector<int>` to `vector<int64_t>`, while it is still `vector<int>` in the script `gen_doc.py`. This patch is to update `gen_doc.py` for this change.

### CODE REVIEW

File: src/builder/op_build_table.inc [hunk 1]
```diff
@@ -18,11 +18,13 @@
        ImportNodeOneOut<mlir::ONNXArgMaxOp>(node, 1, 1, {
          {"axis", 0}
         ,{"keepdims", 1}
+        ,{"select_last_index", 0}
```
Comment (doru1004) [inline src/builder/op_build_table.inc]: Was this removed in error by previous patches?

File: src/builder/op_build_table.inc [hunk 2]
```diff
@@ -18,11 +18,13 @@
...
       });
     }else if (OpName == "ArgMin") {
        ImportNodeOneOut<mlir::ONNXArgMinOp>(node, 1, 1, {
          {"axis", 0}
         ,{"keepdims", 1}
+        ,{"select_last_index", 0}
```
Comment (doru1004) [inline src/builder/op_build_table.inc]: Also this?

File: src/builder/op_build_table.inc (hunk 1 above)
Comment (tungld) [reply src/builder/op_build_table.inc]: I guess so. It was automatically added when I re-ran `gen_doc.py`, though it was not related to the changes in this patch.

Comment (doru1004) [reply src/builder/op_build_table.inc]: Cool. I think it's ok to leave it looks to me like it agrees to the spec.

**Comment by chentong319** [comment]
The attribute select_last_index for ArgMax is added in version 12. Our onnx is version 11.

**Comment by doru1004** [comment]
@chentong319 thanks for the feedback. Sounds good. Let's wait until your patch lands and then we can drop this patch if it is not needed.

**Comment by doru1004** [comment]
Is this patch still needed now that we have moved to a new way of handling attributes? @chentong319

**Comment by chentong319** [comment]
The change of attribute type is already there.
The change of attribute for ArgMax will be automatically generated when the ONNX is updated.  I will follow up on this.

**Pull request closed without merging**